### PR TITLE
fix path issues

### PR DESCRIPTION
**Pull request by JosephBARBIERDARNAL**

### CODE REVIEW

File: inst/assets/rmarkdown_template.html [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        margin-bottom: 15px;
+      }
+
+      .logo {
+        height: 40px;
+        width: auto;
+        background-image: url(logo/OMNI_Logo-White_Horizontal.png);
```
Comment (JosephBARBIERDARNAL) [inline inst/assets/rmarkdown_template.html:62]: @tvroylandt do you know how the path is supposed to looks like? I looked at other examples in the package and none of them seem to work. Is it supposed to be relative to the template?

Comment (tvroylandt) [reply inst/assets/rmarkdown_template.html:62]: See https://github.com/rfortherestofus/omni/blob/fix-assets-in-html/inst/rmarkdown/templates/html_rmarkdown/skeleton/template.html

Comment (JosephBARBIERDARNAL) [reply inst/assets/rmarkdown_template.html:62]: thanks!

**Comment by JosephBARBIERDARNAL** [comment]
@dgkeyes I think that's good now! I'll let you review it and let me know if any adjustments need to be made.

**Comment by dgkeyes** [comment]
Thanks. This looks good to me. I'll go ahead and merge.

@OskaratOmni please go ahead and reinstall and then let us know if you have any issues.

**Pull request merged**